src/app/pages/HomePage/components/Banner/Banner.tsx

```diff
@@ -1 +1,28 @@
-export {}
\ No newline at end of file
+import "./Banner.scss";
+import backgroundimage from "../../../../../assets/images/backgroundImage.png";
+import arrow from "../../../../../assets/icons/Arrow.svg";
+
+export const Banner = () => {
+  return (
+    <>
+      <section className="banner">
+        <div className="background">
+          <img src={backgroundimage} alt="background image" />
+        </div>
+        <div className="intro">
+          <div className="content">
+            <div className="head">Let Your Home Be Unique</div>
```

Comment:
if it is a header, then it's better to use h2 element instead of div
you can consider this in next tasks)